### ISSUE TITLE
False positive: lang or xml:lang attribute missing when the attribute is there

### ISSUE THREAD

**Issue by p4bl1t0** [squizlabs/HTML_CodeSniffer#74]
This is the code: 

```
<html lang="en">
```

When running HTML_CodeSniffer auditing WCAG 2.0 AA, the error show is: 

```
The html element should have a lang or xml:lang 
attribute which describes the language of the 
document.
```

This issue appeared when running in http://a11yproject.com/ website.

Tested in Firefox 22 and in Chrome 28 on Windows XP
